tests/client/test_client.py

```diff
@@ -52,6 +52,10 @@ def test_set_authentication_token(self):
         resp = client.set_authentication_token(token)
         assert resp == {"valid": True}
 
+        # can also access regular Django views that are private
```

Comment:
nice! can you also write a check prior to setting the token that an anon going to the same url results in a redirect to login page?